piccolo_admin/endpoints.py

```diff
@@ -680,7 +680,7 @@ def __init__(
 
         private_app.add_route(
             path="/change-password/",
-            route=change_password(
+            route=change_password(  # type: ignore
```

Comment:
I'm pretty sure the Starlette type annotations are wrong here - we're just passing in a `HTTPEndpoint`.